src/openrct2/ride/VehicleData.cpp

```diff
@@ -885,8 +887,11 @@ const int32_t AccelerationFromPitch[] = {
      416016,    // 57 Inverting Loop Down 42.5
      546342,    // 58 Inverting Loop Down 60
     -110424,    // 59 Slope Up Spiral Lift Hill
+     110424,    // 60 Slope Down Spiral Lift Hill
 };
 
+static_assert(std::size(AccelerationFromPitch) == EnumValue(VehiclePitch::count));
```

Comment:
Can we spin off this fix into its own PR? The diff is quite largely and that would make it easier to manage.

Also, was the code simply reading out of bounds before?

Reply:
it was not reading out of bounds, I added it to make it easier to find and debug issues when adding new pitch values. I can spin these checks into their own PR.